sdk/python/feast/feature_store.py

```diff
@@ -1255,6 +1255,20 @@ def materialize_incremental(
         feature_views_to_materialize = self._get_feature_views_to_materialize(
             feature_views
         )
+
+        if getattr(self.config.online_store, "lazy_table_creation", False):
+            # feature_views_to_delete = self._get_feature_views_to_delete()
+            # don't delete any tables for now
+
+            self._get_provider().update_infra(
```

Comment:
Quick note here: If we use "expedia" as the provider, it basically doesn't do the update and simply throws a message saying these tables will be deleted etc. If you use anything else, this code would work. You'll need to add a validation to check which provider it uses, or maybe update the code in ExpediaProvider to support the infra creation when using the ScyllaDB Online Store creation.

Reply:
Thanks for catching this. I've made changes that I think are now a good way of handling the expedia provider.